### PR TITLE
Hotfix Release v1.0.8 - NTGR-753 - fixed pagination logic

### PR DESCRIPTION
**Pull request by Ndibui**
This PR fixes the pagination logic in `Issuance Logs` page.

### CODE REVIEW

File: includes/helpers/class-accredible-learndash-admin-table-helper.php [hunk 1]
```diff
@@ -430,14 +431,16 @@ class="button accredible-button-outline-natural accredible-button-small"
...
 		 * Resolves href attribute for pagination.
 		 *
+		 * @param string $page page.
+		 *
 		 * @param string $page_num page num used passed to href.
 		 *
 		 * @return string
 		 */
-		private static function get_pagination_href( $page_num ) {
+		private static function get_pagination_href( $page, $page_num ) {
```
Comment (octaviorequena) [inline includes/helpers/class-accredible-learndash-admin-table-helper.php:440]: I think the order of these parameters are incorrect, when you're calling the method, the `page` is the second parameter

**Review by octaviorequena — CHANGES_REQUESTED** [review]
I have left you a comment regarding the new parameters for the method `get_pagination_href`, please can you take a look?

File: includes/helpers/class-accredible-learndash-admin-table-helper.php (hunk 1 above)
Comment (Ndibui) [reply includes/helpers/class-accredible-learndash-admin-table-helper.php:440]: I have updated the calls to this function to have this order.

**Review by octaviorequena — APPROVED** [review]

**Review by kei178 — CHANGES_REQUESTED** [review]
@Ndibui - Looks good. Could you also update the following version info as the [release instructions](https://accredible.atlassian.net/wiki/spaces/PD/pages/3122724888/LearnDash+-+Release+a+Version)?

- readme.txt
  - Add a new change log
  - Update `Stable tag`
- accredible-learndash-add-on.php
  - Update `Version`
  - Update `define( 'ACCREDIBLE_LEARNDASH_VERSION', 'NEW_VERSION' );`

After approved, this branch will also need to be merged into `develop` too.

**Review by kei178 — APPROVED** [review]
LGTM 💯

**Pull request merged**